packages/yarnpkg-libui/sources/components/Pad.tsx

```diff
@@ -0,0 +1,12 @@
+import chalk                  from 'chalk';
+import {Text}                 from 'ink';
+import React, {memo, useMemo} from 'react';
+
+export interface PadProps {
+  length: number
+  active: boolean
+}
+export const Pad = memo(({length, active}: PadProps) => {
+  const text = useMemo(() => chalk.underline(` `.padEnd(length, ` `)), [length, active]);
```

Comment:
Here for example the second `useMemo` doesn't serve practical purpose since it you already wrapped `Pad` in `memo`. But generally speaking I'm not convinced we should use it at all unless we have a concrete bottleneck 🤔